core/lib/src/main/java/org/opennms/core/utils/InetAddressUtils.java

```diff
@@ -1,8 +1,8 @@
 /*******************************************************************************
  * This file is part of OpenNMS(R).
  *
- * Copyright (C) 2007-2014 The OpenNMS Group, Inc.
- * OpenNMS(R) is Copyright (C) 1999-2014 The OpenNMS Group, Inc.
+ * Copyright (C) 2007-2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
```

Comment:
you don't have to change the copyright if you're not changing the rest of the file, IMHO

Reply:
not intentionally :)
I guess those were open in IDE for some reason and I thought I did a change in those files therefore changed the date